packages/store/src/index.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * @import {MustMatch} from '@agoric/internal';
+ */
+import { mustMatch as typelessMustMatch } from '@endo/patterns';
```

Comment:
Is a layering problem causing this duplication, or could all uses of this one use the @agoric/internal one instead?

Reply:
`@agoric/store` is actually lower than `@agoric/internal`, unfortunately

Reply:
I see. Could it be moved and exported from @agoric/store for now? As a permanent home, that would be weird. But if it's ultimate destination is @endo/patterns, well, @agoric/store still does deprecated reexporting of many things from @endo/patterns, since patterns lived there first.

I leave this to your esthetics. Your status quo is not doing an offensive amount of duplication; just a tiny amount.

Reply:
I'll do that before merging, so I won't merge tonight